File: OneDrive/Desktop/Code/CS50/Python/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    height = get_height()
    width = height - 1
    i = 0
    j = 0

    while (width != 0):
        for j in range(width):
            print(" ", end="")
        for i in range(height - width):
            print("#", end="")
        print("  ", end="")
        for i in range(height-width):
            print("#", end="")
        print()
        width -= 1
    for i in range(height):
        print("#", end="")
    print("  ", end="")
    for i in range(height):
        print("#", end="")
    print()


def get_height():
    while True:
        n = get_int("Height: ")
        if n > 0 and n < 9:
            return n


main()

File: OneDrive/Desktop/Code/CS50/C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get height of grid
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //Print left grid
    for (int i = 0; i < n; i++)
    {
        for (int j = i + 1; j < n; j++)
        {
            printf(" ");
        }
        for (int g = n + i + 1; g > n; g--)
        {
            printf("#");
        }
        //Print right grid
        printf("  ");

        for (int g = n + i + 1; g > n; g--)
        {
            printf("#");
        }
        printf("\n");
    }
}
